modules/checking-mco-status-certs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="checking-mco-status-certs_{context}"]
+= Viewing and interacting with certificates
+
+The following certificates are handled in the cluster by the Machine Config Controller (MCC) and can be found in the `ControllerConfig` resource:
+
+* `/etc/kubernetes/kubelet-ca.crt`
+* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
+* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`
+
+You can get information about these certificates, including the underyling bundle the certificate comes from, and the signing and subject data.
+
+.Procedure
+
+* Get detailed certificate information by running the following command:
++
+[source,terminal]
+----
+$ oc describe controllerconfig/machine-config-controller
+----
++
+.Example output
++
+[source,text]
+----
+"controllerCertificates": [
+                   {
+                       "bundleFile": "KubeAPIServerServingCAData",
+                       "signer": "<signer_data1>",
+                       "subject": "CN=openshift-kube-apiserver-operator_node-system-admin-signer@168909215"
+                   },
+                   {
+                       "bundleFile": "RootCAData",
+                       "signer": "<signer_data2>",
+                       "subject": "CN=root-ca,OU=openshift"
+                   }
+                ]
+----
+
+* Get a simpler version of the information found in the ControllerConfig by checking the machine config pool status using the following command:
++
+[source,terminal]
+----
+$ oc get mcp master -o yaml | grep certExpirys -B1 -A15
+----
++
+.Example output
++
+[source,text]
+----
+status:
+  certExpirys:
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=admin-kubeconfig-signer,OU=openshift
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=kube-csr-signer_@1689585558
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=kubelet-signer,OU=openshift
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=kube-apiserver-to-kubelet-signer,OU=openshift
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=kube-control-plane-signer,OU=openshift
+----
++
+This method is meant for {product-title} applications that already consume machine config pool information.
+
+You can check which certificates are on the nodes by looking at `/etc/docker/cert.d`.
```

Comment:
In  `/etc/docker/cert.d` we will find the "image registry" certificates and the "image registry user bundle" certificate.

The rest of the certificates have their own path, that we mention at the beginning of the document

* `/etc/kubernetes/kubelet-ca.crt`
* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`


Please, @cdoern  could your confirm that?

Reply:
Yes, sorry I should have clarified that. The data inside of /etc/docker/certs.d is solely image registry certificates.